go.mod

```diff
@@ -197,6 +196,7 @@ replace (
 	github.com/cockroachdb/datadriven => github.com/cockroachdb/datadriven v0.0.0-20200714090401-bf6692d28da5
 	github.com/cockroachdb/errors => github.com/cockroachdb/errors v1.2.4
 	github.com/cockroachdb/logtags => github.com/cockroachdb/logtags v0.0.0-20190617123548-eb05cc24525f
+	github.com/codegangsta/inject => github.com/codegangsta/inject v0.0.0-20150114235600-33e0aa1cb7c0
```

Comment:
we also don't depend on this at HEAD today, and this looks unmaintained... what's pulling this in?

Reply:
hmm I missed this because CI is happy with the state of vendor and there are no vendor changes here.

Reply:
[discussed offline: this affects dependency resolution even if it's indirect and not depended on in our binaries]

Reply:
@liggitt : For both the comments regarding new deps, to add to it what @BenTheElder said specifically, it is a indirect dependency from https://github.com/auth0/go-jwt-middleware/blob/master/go.mod#L6-L8

Reply:
It looks like  https://github.com/auth0/go-jwt-middleware only uses that for examples/martini-example/main.go. Opened https://github.com/auth0/go-jwt-middleware/pull/94 to isolate examples to their own go.mod file so that downstreams of the library don't pick them up.

Reply:
That is a great idea @liggitt :) Love it!!

Reply:
https://github.com/auth0/go-jwt-middleware/releases/tag/v1.0.1 is usable now and drops these extraneous deps

Reply:
Just pushed the changes to bump `go-jwt-middleware` Will wait for job runs to go green and then ping back :)